parsers/nmparser/get_model_output.go

```diff
@@ -95,6 +95,12 @@ func GetModelOutput(filePath string, verbose, noExt, noGrd, noCov, noCor, noShk
 }
 
 func setDefaultValues(results *ModelOutput, etaCount, epsCount int) {
+	// method name
+	for i := range results.ParametersData {
+		if len(results.ParametersData[i].Method) == 0 {
+			results.ParametersData[i].Method = fmt.Sprintf("Method%v", i+1)
```

Comment:
this might be incorrect accidentally - if this came back as Method1 due to a default, Method1 is "FOCE". IMO the default method should be "METHOD NOT DETECTED" or something along those lines